server/sonar-server/src/main/java/org/sonar/server/organization/ws/RemoveMemberAction.java

```diff
@@ -0,0 +1,96 @@
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.organization.ws;
+
+import org.sonar.api.server.ws.Request;
+import org.sonar.api.server.ws.Response;
+import org.sonar.api.server.ws.WebService;
+import org.sonar.api.server.ws.WebService.NewController;
+import org.sonar.db.DbClient;
+import org.sonar.db.DbSession;
+import org.sonar.db.organization.OrganizationDto;
+import org.sonar.db.organization.OrganizationMemberDto;
+import org.sonar.db.permission.OrganizationPermission;
+import org.sonar.db.user.UserDto;
+import org.sonar.server.exceptions.BadRequestException;
+import org.sonar.server.user.UserSession;
+
+import static java.lang.String.format;
+import static org.sonar.server.organization.ws.OrganizationsWsSupport.PARAM_LOGIN;
+import static org.sonar.server.organization.ws.OrganizationsWsSupport.PARAM_ORGANIZATION;
+import static org.sonar.server.ws.KeyExamples.KEY_ORG_EXAMPLE_001;
+import static org.sonar.server.ws.WsUtils.checkFound;
+import static org.sonar.server.ws.WsUtils.checkFoundWithOptional;
+
+public class RemoveMemberAction implements OrganizationsWsAction {
+  private final DbClient dbClient;
+  private final UserSession userSession;
+
+  public RemoveMemberAction(DbClient dbClient, UserSession userSession) {
+    this.dbClient = dbClient;
+    this.userSession = userSession;
+  }
+
+  @Override
+  public void define(NewController context) {
+    WebService.NewAction action = context.createAction("remove_member")
+      .setDescription("Remove a member from an organization.<br>" +
+        "Requires 'Administer System' permission on the specified organization.")
+      .setSince("6.4")
+      .setPost(true)
+      .setInternal(true)
+      .setHandler(this);
+
+    action
+      .createParam(PARAM_ORGANIZATION)
+      .setDescription("Organization key")
+      .setRequired(true)
+      .setExampleValue(KEY_ORG_EXAMPLE_001);
+
+    action
+      .createParam(PARAM_LOGIN)
+      .setDescription("User login")
+      .setRequired(true)
+      .setExampleValue("ray.bradbury");
+  }
+
+  @Override
+  public void handle(Request request, Response response) throws Exception {
+    String organizationKey = request.mandatoryParam(PARAM_ORGANIZATION);
+    String login = request.mandatoryParam(PARAM_LOGIN);
+
+    try (DbSession dbSession = dbClient.openSession(false)) {
+      OrganizationDto organization = checkFoundWithOptional(dbClient.organizationDao().selectByKey(dbSession, organizationKey), "Organization '%s' is not found",
+        organizationKey);
+      UserDto user = checkFound(dbClient.userDao().selectByLogin(dbSession, login), "User '%s' is not found", login);
+
+      userSession.checkPermission(OrganizationPermission.ADMINISTER, organization);
+
+      OrganizationMemberDto organizationMember = dbClient.organizationMemberDao().select(dbSession, organization.getUuid(), user.getId())
```

Comment:
On my side I won't fail, because as a user I want to do something that is already done => fine.